src/event.rs

```diff
@@ -47,7 +47,7 @@ impl Event {
     /// spending plan is valid.
     pub fn verify(&self) -> bool {
         match *self {
-            Event::Transaction(ref tr) => tr.verify(),
+            Event::Transaction(ref tr) => tr.verify_plan(),
```

Comment:
This change shouldn't be necessary. Can you revert that and report back any consequences.

Reply:
I changed to verify_sig(), but did you mean it should be verify_sig() && verify_plan(). Anatoly removed the one that does both, but I can add it back.